### PR TITLE
Merge zed into antelope

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: etc/kayobe/pulp-host-image-versions.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 # Overcloud host image versioning tags
 # These images must be in SMS, since they are used by our AIO CI runners
-stackhpc_rocky_9_overcloud_host_image_version: "yoga-20230929T133006"
-stackhpc_ubuntu_jammy_overcloud_host_image_version: "yoga-20230609T120720"
+stackhpc_rocky_9_overcloud_host_image_version: "zed-20231013T123933"
+stackhpc_ubuntu_jammy_overcloud_host_image_version: "zed-20231013T123933"
```
Comment (Alex-Welsh) [inline etc/kayobe/pulp-host-image-versions.yml:5]: I'm aware these need to change, building antelope host images is on the todo list

File: terraform/aio/README.rst [hunk 2]
```diff
@@ -84,7 +84,7 @@ Generate Terraform variables:
    cat << EOF > terraform.tfvars
    ssh_public_key = "id_rsa.pub"
    aio_vm_name = "kayobe-aio"
-   aio_vm_image = "Rocky9"
+   aio_vm_image = "overcloud-rocky-9-zed-20231013T123933"
```
Comment (Alex-Welsh) [inline terraform/aio/README.rst:87]: ditto

File: doc/source/operations/upgrading.rst [hunk 3]
```diff
@@ -0,0 +1,926 @@
...
+
+After preparation is complete, the remaining phases may be completed in any
+order, however the order specified above allows for completing as much as
+possible before the user-facing overcloud upgrade. It is not recommended to
+keep different parts of the system on different releases for extended periods
+due to the need to maintain and use separate local Kayobe environments.
+
+.. NOTE(upgrade): Update these notable changes for the current release.
+
+Notable changes in the |current_release| Release
```
Comment (markgoddard) [inline doc/source/operations/upgrading.rst:32]: This section can be updated separately.

**Review by markgoddard — APPROVED** [review]

File: doc/source/operations/upgrading.rst (hunk 3 above)
Comment (Alex-Welsh) [reply doc/source/operations/upgrading.rst:32]: ack

**Pull request merged**